bigquery/tests/unit/model/test_model.py

```diff
@@ -21,6 +21,8 @@
 import google.cloud._helpers
 from google.cloud.bigquery_v2.gapic import enums
 
+KMS_KEY_NAME = "projects/1/locations/global/keyRings/1/cryptoKeys/1"
```

Comment:
I know this is a unit test, but we're supposed to discourage use of `global` location. Let's change this to `us`.